define_strong_self.m

```diff
@@ -0,0 +1,10 @@
+// Define Strong Self
+// 
+//
+// IDECodeSnippetCompletionPrefix: vokStrongSelf
+// IDECodeSnippetCompletionScopes: [CodeBlock]
+// IDECodeSnippetIdentifier: 1A45777F-3BA7-409C-BFA2-7B8FBCF20C58
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+typeof(weakSelf) strongSelf = weakSelf;
```

Comment:
I often put the `__strong` qualifier in here, not because it's necessary, but because it makes it more explicit.  :shrugs: